src/Components/header.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+export const Container = styled.header`
+  max-width: 1300px;
+  width: 80%;
```

Comment:
I'd rather use `width: 100%` here with a fixed horizontal padding. I guess it's 80px in the design. Using `width: 80%` will make it hard to align other elements with the header and might look strange on smaller devices. But that's just my guess. 

I usually see `width: 100%` in combination with a `max-width` and some padding. You can also check the example implementation.

Reply:
I have fixed it now. These are some really helpful insights. 
I'm absolutely enjoying this course!